playbooks/conferences.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Conferences
+description: Attending or speaking at a conference
+---
+
+# Conferences
+
+We encourage engineers to attend and speak at conferences. It is a great way to stay up-to-date with the industry,
+be inspired by challenges others are facing, develop personal career, and connect with the community. Here are some
+guidelines:
+
+## Attending Conferecens
+
+- We aim for each engineer to be able to attend a conference every year, subject to budget. The conference should
+  be related to individual's role or Artsy's challenges/technologies.
+- Please be mindful when expensing your travel. A good rule of thumb is to spend the money like your own. Refer to
+  [Atlas][travel-and-accommodations-on-atlas] for more details and rate card.
+- It's strongly recommended writing up take-aways and sharing with the team upon return, e.g. via [Lunch and
+  Learn][lunch-and-learn], [Show and Tell][show-and-tell], or a blog post.
+
+## Speaking at Confereces
+
+- We encourage engineers to speak at conferences, and budget-wise the expense gets higher priority.
+- If you need help preparing a conference talk, stop by [Ash][ash]'s weekly Writing Office Hours.
+- It's also recommended sharing the talk and lessons with the team.
```

Comment:
I guess I should L&L on GraphQL errors! 😄

Reply:
I'd love to see it! 🍻